OpenSim/Common/Exception.h

```diff
@@ -67,14 +110,31 @@ class OSIMCOMMON_API Exception  : public std::exception {
     std::string _file;
     /** Line number at which the error occurred. */
     int _line;
+    /** Delimiter/separator used to separate different parts of the message.  */
+    static const std::string _msgSeparator;
```

Comment:
> This PR is meant to be non-invasive. 

I don't know what that means! I think it is a very good thing for you to start making wider changes in OpenSim and this is an easy one. This PR is the first modification to the Exception class in a long time; if those fields aren't actually protected this is definitely the time to fix them. No one is going to open a PR just to do that.